solutions/alt-tag-generator/README.md

```diff
@@ -6,7 +6,7 @@ This project will generate relevant alt text for images using AI.
 
 ## How it works
 
-It uses an ML modal from Salesforce called [BLIP](https://github.com/salesforce/BLIP) on [Replicate](https://replicate.com/) to generate relevant alt text for images. You can feed the Next.js API route an image as a query param and it will return a one sentence description of that image.
+It uses an ML modal from Salesforce called [BLIP](https://github.com/salesforce/BLIP) on [Replicate](https://replicate.com/) to generate relevant alt text for images. You can feed the Vercel Edge Function route an image as a query param and it will return a one sentence description of that image.
```

Comment:
It's an API Route still, just running at edge

Reply:
<picture data-single-emoji=":ack:" title=":ack:"><img class="emoji" width="20" height="auto" src="https://single-emoji.vercel.app/api/emoji/eyJhbGciOiJkaXIiLCJlbmMiOiJBMjU2R0NNIn0..DpGOOYdHFuuBV-7V.TnLbLA0cTSGBNbgdBYa2MMSzbslFfKMi_cwyJc0jhcSkr8bBEXX4gU_liVv6RhJV8-7YSsyY9mXj7m1OHhnE-xZQT1dbTrjO8vii4hxOIWL1UJTz5g.d5iDu9RpJfJvmrnrTyAxJQ" alt=":ack:" align="absmiddle"></picture>